src/Views/gagepage.html

```diff
@@ -224,7 +224,7 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                     <div ng-if="!vm.chartConfig" class="panel panel-default reportSpinner">
                         &nbsp;<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>
                     </div>
-                    <center>
+                    <!-- <center>
```

Comment:
Remember to uncomment for the final pull request 

```suggestion
                    <center>
```